README.md

```diff
@@ -4,6 +4,19 @@ This is a version manager for [RGBDS](https://github.com/gbdev/rgbds). It is a p
 
 \* Bash script, at the moment. Compatibility with other shells is not guaranteed.
 
+## Quickstart (Debian)
+```sh
+sudo apt install -y libpng-dev pkg-config build-essential bison git curl
+sudo sh -c 'curl https://raw.githubusercontent.com/gbdev/rgbenv/master/rgbenv > /usr/local/bin/rgbenv'
+sudo chmod +x /usr/local/bin/rgbenv
+yes | rgbenv install 0.7.0
+rgbenv use 0.7.0
```

Comment:
Perhaps splitting "install" then "use" instructions in two blocks?

Reply:
Or three blocks: getting rgbenv, installing a version, using a version?

Reply:
Yeah, sounds like a good idea.